### PR TITLE
remove jcenter

### PR DESCRIPTION
**Pull request by dulmandakh**
## Summary

jcenter is read-only now, and newer versions of dependencies will be published to either MavenCentral or Jitpack. This PR removes jcenter to avoid future issues, then uses MavenCentral and Jitpack as replacement. Current flipper depends on Stetho version that is not available on MavenCentral, so had to exclude and bump the version.

Both Gradle and Buck successfully download all the dependencies.

## Changelog

[Android] [Changed] - Remove jcenter

## Test Plan

rn-tester builds and runs as expected.

### CODE REVIEW

**Comment by dulmandakh** [comment]
@mdvacca Please review and merge

**Comment by ShikaSD** [comment]
Hey, thanks for updating this!

> then uses MavenCentral and Jitpack as replacement

Do you know which dependencies are available only on Jitpack?

**Comment by dulmandakh** [comment]
@ShikaSD i don't remember, there was a few. Most dependencies moved to MavenCentral, but i thought that having jitpack would ease migration for developers.

**Comment by dulmandakh** [comment]
@ShikaSD removed jitpack from buck.

File: build.gradle.kts [hunk 1]
```diff
@@ -38,16 +33,13 @@ allprojects {
         mavenLocal()
         google()
         mavenCentral()
-        jcenter {
-          content {
-            includeModule("com.facebook.yoga", "proguard-annotations")
-            includeModule("com.facebook.fresco", "stetho")
-          }
+        maven {
```
Comment (ShikaSD) [inline build.gradle.kts:36]: Hm, I believe we also don't need jitpack here
Only FB specific ones were remaining, according to the list above, and they should be available on Maven Central

Comment (dulmandakh) [reply build.gradle.kts:36]: Yes, I can remove it but tomorrow

**Pull request closed without merging**